plugins/sla-email/src/main/java/com/thinkbiganalytics/metadata/sla/config/EmailServiceLevelAgreementSpringConfiguration.java

```diff
@@ -59,26 +59,35 @@ public EmailConfiguration emailConfiguration() {
     @Bean(name = "slaEmailSender")
     public JavaMailSender javaMailSender(@Qualifier("slaEmailConfiguration") EmailConfiguration emailConfiguration) {
         JavaMailSenderImpl mailSender = new JavaMailSenderImpl();
-        Properties mailProperties = new Properties();
-        mailProperties.put("mail.smtp.auth", emailConfiguration.isSmtpAuth());
-        mailProperties.put("mail.smtp.starttls.enable", emailConfiguration.isStarttls());
+
+        Properties mailProperties = mailSender.getJavaMailProperties();
+        mailProperties.put("mail.smtp.auth", StringUtils.defaultIfBlank(emailConfiguration.getSmtpAuth(),"true"));
+        mailProperties.put("mail.smtp.starttls.enable", StringUtils.defaultIfBlank(emailConfiguration.getStarttls(),"true"));
```

Comment:
I think the previous value for these properties was 'false', so this may introduce additional upgrade steps. Could we keep the default value as 'false' for now?

Reply:
Sure. I thought that it might be simpler for future deployments, as most email providers require TLS